src/node.js

```diff
@@ -34,6 +67,35 @@ define(function () {
     return node.parentNode.removeChild(node);
   }
 
+  function getAncestor(node, rootElement, nodeFilter) {
+    function isTopContainerElement (element) {
+      return rootElement === element;
+    }
+    // TODO: should this happen here?
+    if (isTopContainerElement(node)) {
+      return;
+    }
+
+    var currentNode = node.parentNode;
+
+    // If it's a `contenteditable` then it's likely going to be the Scribe
+    // instance, so stop traversing there.
+    while (currentNode && ! isTopContainerElement(currentNode)) {
+      if (nodeFilter(currentNode)) {
+        return currentNode;
+      }
+      currentNode = currentNode.parentNode;
+    }
+  }
+
+  function nextAll(node) {
```

Comment:
What do you think of nextSiblings? More idiomatic, though using inflections might throw people off ("I did not see the s").